spec/cypress/integration/admin/SchoolManagment.feature

```diff
@@ -26,8 +26,9 @@ Feature: Admin user managing schools
     And the page should be accessible
     And percy should be sent snapshot
 
+  @focus
   Scenario: Viewing a list of schools
-    Then the table should have 20 rows
+    Then the table should have 12 rows
```

Comment:
This isn't directly related to this PR, but I was concerned that it was testing for 20 rows when there were actually 23 schools and it was just testing the pagination.